### ISSUE TITLE
Set up event to share about PyMC v4.0 release

### ISSUE THREAD

**Issue by reshamas** [pymc-devs/video-timestamps#6]
Not sure where the best place to open up the issue is.

(Tangential thought: wondering if we should rename this repo "community" or something).

Back to the topic:  
- Set a date and speaker to discuss in more details the v4.0 release
- Once a date is set, set up Meetup event
- Do social media card and promotion

cc:  @twiecki @cluhmann @OriolAbril

**Comment by OriolAbril** [comment]
> (Tangential thought: wondering if we should rename this repo "community" or something).

I don't think so. If that is to indicate that public discussion about community topics should happen in the issues here I'd vote against it. IMO community discussions should happen on discourse to engage with the community. 

For example: https://discourse.pymc.io/t/proposal-for-website-design/9483 (with a user weighing in) or https://discourse.pymc.io/t/proposal-for-website-design/9483 (even though only core contributors participated here). We can discuss renaming or creating tags and categories to make it more obvious.

> Set a date and speaker to discuss in more details the v4.0 release

It might be more work to plan, but I think it would be great to have multiple speakers, each covering some of the new features
